.circleci/config.yml

```diff
@@ -37,9 +35,10 @@ push_images: &push_images
         curl -fsSL https://download.docker.com/linux/static/stable/x86_64/docker-${DOCKER_VERSION}.tgz | tar xz -C /usr/bin --strip-components 1
       fi
 
+      # Check, if docker images declared by jobs have actually been built
       IMAGES_TO_PUSH=""
-      for image in ${DOCKER_IMAGES_TO_PUSH}; do
-        if [ ! -z "$(docker images -q $image)" ]; then
+      for image in ${DOCKER_IMAGES}; do
```

Comment:
Note that I'm using the DOCKER_IMAGES variable, which is set in individual jobs to the docker image name produced by that job see e.g. https://github.com/syndesisio/syndesis/pull/5786/files#diff-1d37e48f9ceff6d8030570cd36286a61R88 There's no need to maintain a separate list of images in such a case. I'm still checking, if images have been built, but not sure, if there's a need for that and maybe it's better to fail the build, if an image declared by a job is missing.